cmd/meroxa/turbine_cli/utils.go

```diff
@@ -242,6 +243,9 @@ func GoInit(l log.Logger, appPath string, skipInit, vendor bool) error {
 	skipLog := "skipping go module initialization\n\tFor guidance, visit " +
 		"https://docs.meroxa.com/beta-overview#go-mod-init-for-a-new-golang-turbine-data-application"
 	goPath := os.Getenv("GOPATH")
+	if goPath == "" {
+		goPath = build.Default.GOPATH
```

Comment:
probably good to log here that the default is being used here because GOPATH is not set

Reply:
agreed - would be nice to display a log when it happens

Reply:
Solved.

Reply:
did you test this manually?  i don't think that's compatible with the spinner

Reply:
![Screenshot from 2022-08-11 12-08-21](https://user-images.githubusercontent.com/12731615/184220000-0b91abd0-b957-49e2-a467-1aefaf35e9f8.png)